packages/libs/eda/src/lib/map/analysis/MapAnalysis.tsx

```diff
@@ -900,6 +898,9 @@ function MapAnalysisImpl(props: ImplProps) {
                     defaultViewport={defaultViewport}
                     // for multiple markers cancelation of selection only
                     setSelectedMarkers={setSelectedMarkers}
+                    onMapClick={setIsSidePanelExpanded}
+                    onMapDrag={setIsSidePanelExpanded}
+                    onMapZoom={setIsSidePanelExpanded}
```

Comment:
Instead of passing the in the `useState` setter function, use a wrapper function that calls it:

```ts
const closePanel = useCallback(() => setIsSidePanelExpanded(false), []);
```

```suggestion
                    onMapClick={closePanel}
                    onMapDrag={closePanel}
                    onMapZoom={closePanel}
```